project-addons/flask_middleware_connector/models/account_invoice.py

```diff
@@ -0,0 +1,19 @@
+from openerp import models, fields, api
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
+
+    orders = fields.Char('Orders', compute='get_orders', readonly=True, store=True)
+
+    @api.multi
+    @api.depends('invoice_line')
```

Comment:
¿Es necesario que sea store=True? es que si tiene que guardarlo para cualquier cambio que se haga en las lineas de factura es bastante costoso.

Reply:
En principio si, ya que lo necesitamos para exportarlo a flask, ¿habría forma de no guardarlo en bbdd y aun así poder exportarlo?

Reply:
Sí, lo calcula en el momento de hacer el get, que lo estáis haciendo cuando se envía a flask la factura, si es así sólo es mejor ponerle store=False, probadlo pero creo que os va a seguir funcionando igual.